server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponseCreateDTO.java

```diff
@@ -11,12 +11,16 @@
 public class PulseResponseCreateDTO {
 
     @NotNull
-    @Schema(required = true, description = "date for submissionDate")
-    private LocalDate submissionDate;
+    @Schema(required = true, description = "integer value of internal score")
+    private Integer internalScore;
 
     @NotNull
-    @Schema(required = true, description = "date for updatedDate")
-    private LocalDate updatedDate;
+    @Schema(required = true, description = "integer value of external score")
```

Comment:
I think we should make external scores and feelings nullable. That would mean changing the field below this as well. We definitely want both feelings to be nullable. Those are comments. The only thing that I think we want to require is a score on the "internal" (about work) feelings.

Reply:
Adjusted + tests

Reply:
Almost what I meant... There are three fields that should be nullable: internalFeelings, externalScore, and externalFeelings

internalScore should not be nullable.

Reply:
@mkimberlin This has been done.